packages/strapi-design-system/src/v2/Link/Link.js

```diff
@@ -33,9 +33,9 @@ const IconWrapper = styled(Box)`
   display: flex;
 `;
 
-export const Link = React.forwardRef(({ children, href, disabled, startIcon, endIcon, ...props }, ref) => {
+export const Link = React.forwardRef(({ children, href, disabled, startIcon, endIcon, isExternal, ...props }, ref) => {
```

Comment:
Some comments on `Link` v2:
- could you please fix the outline + icon color issue we can see in SimpleMenu v2 with:
```js
const LinkWrapper = styled(BaseLink)`
  ...

  color: ${({ disabled, theme }) => (disabled ? theme.colors.neutral600 : theme.colors.primary600)};

  svg path {
    transition: fill 150ms ease-out;
    fill: currentColor;
  }

  ...

  ${buttonFocusStyle}; <=== to remove

  &:focus-visible {
    border-radius: ${({ theme }) => theme.borderRadius};
  }
`;
```